### DISCUSSION TITLE
2021.06.10

### DISCUSSION THREAD

**Discussion by Settpark** [malaheaven/issue-tracker#32] [General]
iOS: 오늘 이슈 작성 뷰 그리고 코드 리팩토링 했습니다. iOS는 오늘 행복합니다.

**Comment by malaheaven** [comment]
### BE: 

로그인기능:
- oauth 구현을 완료했습니다. BE는 오늘도 행복합니다
- 내일은 jwt가 남았습니다 흙흙 행복합니다

**Comment by 17-sss** [comment]
**FE**

- 오늘은 이슈리스트 UI 제작을 시작했습니다
- 했다가 시행착오가... ㅠㅠㅠ
- FE는 행복합니다.. ***Maybe***
